tests/markdown_abbr/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,151 @@
+parsers: ["markdown"]
+printWidth: 80
+proseWrap: "always"
+                                                                                | printWidth
+=====================================input======================================
+*[*em*]: *that's en emphasis*
+*[star]: and that's not an empasis, that's just a star*
+*[that's an empasis*
+**[and that's just bold]: right?**
+*[link](:that's still a link though)
+
+* and
+* that's
+* [a]: list
+
+*Not here though*
+*[raw]:*Actually, any kind  of formatting is ignored here*
+*nor here*
+
+=====================================output=====================================
+*[*em*]: *that's en emphasis*
+
+
+*[star]: and that's not an empasis, that's just a star*
+
+
+*[that's an empasis*
+**[and that's just bold]: right?**
+ \\*[link](:that's still a link though)
+
+- and
+- that's
+- [a]: list
+
+_Not here though_
+
+*[raw]: *Actually, any kind  of formatting is ignored here*
+ _nor here_
+
+================================================================================
+`;
+
+exports[`even.md 1`] = `
+====================================options=====================================
+parsers: ["markdown"]
+printWidth: 80
+proseWrap: "always"
+                                                                                | printWidth
+=====================================input======================================
+An even test to make sure *we don't get caught by the emphasis*, or even _these ones_
+
+*[MD]:Markdown
+*[CSS]: Cascading Style Sheets
+=====================================output=====================================
+An even test to make sure _we don't get caught by the emphasis_, or even _these
+ones_
+
+*[MD]: Markdown
+
+
+*[CSS]: Cascading Style Sheets
+
+
+================================================================================
+`;
+
+exports[`long.md 1`] = `
+====================================options=====================================
+parsers: ["markdown"]
+printWidth: 80
+proseWrap: "always"
+                                                                                | printWidth
+=====================================input======================================
+*[LONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGABBR] :  Long abbr
+
+*[AMLAEIKOUR]:A much longer abbreviation explanation, it's kind of unneeded really
+*[LONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGABBR]         :                A much longer abbreviation explanation, it's kind of unneeded really
+
+=====================================output=====================================
+*[LONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGABBR]: Long abbr
+*[AMLAEIKOUR]: A much longer abbreviation explanation, it's kind of unneeded really
+*[LONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGABBR]: A much longer abbreviation explanation, it's kind of unneeded really
```

Comment:
Is it allowable for these to break?

```markdown
*[LONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGLONGABBR]:
  A much longer abbreviation explanation, it's kind of unneeded really
```

Reply:
[CommonMark](https://spec.commonmark.org/0.28/) doesn't have a specification for `abbr`. It seems this notation originates from [PHP Markdown](https://michelf.ca/projects/php-markdown/extra/#abbr), and it seems they define broken `abbr` as empty:

>  An abbreviation may also have an empty definition, in which case <abbr> tags will be added in the text but the title attribute will be omitted.

Lookign at the implementations, neither [remark-abbr](https://github.com/zestedesavoir/zmarkdown/blob/master/packages/remark-abbr/src/index.js#L9) nor [markdown-it-abbr](https://github.com/markdown-it/markdown-it-abbr/blob/master/index.js#L43) support breakable `abbr`